Miniforge3/construct.yaml

```
@@ -1,4 +1,4 @@
-{% set version = os.environ.get("MINIFORGE_VERSION", "25.1.1-2") %}
+{% set version = os.environ.get("MINIFORGE_VERSION", "25.1.1-3") %}
```

```suggestion
{% set version = os.environ.get("MINIFORGE_VERSION", "25.1.1-2") %}
```

I don't think there is a need to bump, we won't be releasing this.

---

I'll roll it back to 2.